context/app/static/js/components/Detail/Dataset/Dataset.jsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+
 import { StyledDivider } from './style';
 import ProvTabs from '../ProvTabs';
 import Summary from '../Summary';
 import Attribution from '../Attribution';
 import Protocol from '../Protocol';
 import MetadataTable from '../MetadataTable';
-import FileTable from '../FileTable';
 import Visualization from '../Visualization';
 import DetailLayout from '../DetailLayout';
+import Files from '../Files';
+
+// TODO use this context for components other than FileBrowser
+import DatasetContext from './context';
```

Comment:
It seems like this context should be located at a common place relative to where it is used, for example `Detail/contexts.js`